paynow/model.py

```diff
@@ -157,32 +159,37 @@ class Payment:
     def __init__(self, reference, auth_email):
         self.reference = reference
         self.auth_email = auth_email
+        # auto-check to ensure clear list
+        self.clearCart()
```

Comment:
Can you apply snake casing here to self.clear_cart ?
I see there is no need for developers to manually call the `self.clear_cart`  , I guess this could become self.__clear_cart unless it's an intention to expose the method

Reply:
thanks for that, if i find time i will make proper changes for a proper `PR` to reflect that change